core/build.gradle.kts

```
@@ -27,6 +27,7 @@ kotlin {
     iosX64()
     iosArm32()
     iosArm64()
+    iosSimulatorArm64()
```

I haven't tried targeting Apple silicon yet, but we probably want both `iosSimulatorArm64`, which is the Apple silicon equivalent of `iosX64`, and `macosArm64`, which is the Apple silicon target equivalent of `macosX64`.
```suggestion
    iosSimulatorArm64()
    macosArm64()
```